cobra/util/util.py

```diff
@@ -29,7 +29,8 @@ def show_versions():
     """Print dependency information."""
     print_dependencies("cobra")
 
+
 def is_not_sane(string):
     """Check if a string is sane to be used as an ID for cobra components."""
     return not isinstance(string, string_types) or \
-        len(string) < 1 or string is ' '
\ No newline at end of file
+        len(string) < 1 or string is ' '
```

Comment:
Actually, any space anywhere in an identifier is nasty for us (and optlang will complain for sure). Which we would test with: `any(c.isspace() for c in string)`. `string` is also the name of a Python stdlib package so maybe change it to `identifier`.